### ISSUE TITLE
Add support for @angular/router

### ISSUE THREAD

**Issue by alexpods** [alexpods/angular2-universal-starter#64]

**Comment by glebmachine** [comment]
This issue is about `error TS2307: Cannot find module '@angular/router'.`?

**Comment by alexpods** [comment]
@glebmachine No, the issue is about adding support for angular/router for all environments (browser main thread, web workers and server side).

Thank you for reporting this bug with `error TS2307: Cannot find module '@angular/router'`. I've just fixed it. Pull the latest master. Everything should be ok.

**Comment by glebmachine** [comment]
Aw, go it, thank for bugfix!)
